openedx_webhooks/templates/github_pr_cover_letter.md.j2

```diff
@@ -18,11 +18,15 @@
 If you've been tagged for review, please check your corresponding box once you've given the :+1:.
 - [ ] Code review #1
 - [ ] Code review #2
-- [ ] UI strings/error msgs review
+- [ ] Docs review
 - [ ] UX review
 - [ ] Accessibility review
 - [ ] Product review
 
+# Documentation considerations
+- [ ] Are there user visible changes? If so, list who is affected, and include a pointer to the relevant code.
+- [ ] Should this be indicated in release notes? If so, include a draft description of your changes.
```

Comment:
I also don't want people to think they need to repeat work, so maybe replace "If so, include a draft description of your changes." with "If so, make sure that you included a description"